tests/integrated/test-deployment.R

```diff
@@ -8,7 +8,8 @@ test_that("can create content", {
   expect_equal(cont1$name, cont1_name)
   expect_equal(cont1$title, cont1_title)
 
-  get_cont1 <- test_conn_1$content(guid = cont1$guid)
+  # Use include = NULL because we aren't passing the include in content_create()
+  get_cont1 <- test_conn_1$content(guid = cont1$guid, include = NULL)
```

Comment:
This test failed before I added this, but I could not reproduce it locally for some reason. That said, I think it's fine to do because `content_create()` doesn't result in a fully hydrated content object anyway, it only creates the entity record, and some fields like about the runtime aren't populated until you deploy a bundle. So the content record you get back from that `POST` won't ever be all that useful (except as a source of the guid of the record that was created).